test/config/tslint-extends-default-severity.json

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": [ "./tslint-default-severity-unspecified.json", "./tslint-default-severity-error.json" ],
+  "defaultSeverity": "warning",
+  "rules": {
+    "default-severity-warning": { "severity": "default" }
+  }
+}
```

Comment:
Please add another config that has no defaultSeverity, that extends this one.
The test then checks that rules specified in the newly added extending config get a severity of "error" while all 3 rules coming from this config are warnings.

Reply:
Please suggest the name for this file. I could come up only with `tslint-extends-default-severity-only-in-extended.json`.

Reply:
Just take the name you suggested. I cannot come up with a better one.